<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="0" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2019-10-04T15:37:53Z" name="Default suite" finished-at="2019-10-04T15:38:14Z" duration-ms="20959">
    <groups>
    </groups>
    <test started-at="2019-10-04T15:37:53Z" name="Default test" finished-at="2019-10-04T15:38:14Z" duration-ms="20959">
      <class name="com.Konakart.Testscript.ValidateProductOutcome">
        <test-method is-config="true" signature="startReport()[pri:0, instance:com.Konakart.Testscript.ValidateProductOutcome@4387b79e]" started-at="2019-10-04T21:07:53Z" name="startReport" finished-at="2019-10-04T21:07:53Z" duration-ms="588" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startReport -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.Konakart.Testscript.ValidateProductOutcome@4387b79e]" started-at="2019-10-04T21:07:53Z" name="setup" finished-at="2019-10-04T21:08:12Z" duration-ms="19072" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateproductOutcome(java.lang.String)[pri:0, instance:com.Konakart.Testscript.ValidateProductOutcome@4387b79e]" started-at="2019-10-04T21:08:14Z" name="validateproductOutcome" data-provider="possibleValues" finished-at="2019-10-04T21:08:14Z" duration-ms="0" status="SKIP">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:49)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:115)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:509)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1308)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1036)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1126)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: java.lang.NullPointerException
	at com.Konakart.Utilities.ExcelUtils.ReadWriteExcel(ExcelUtils.java:43)
	at com.Konakart.Testscript.ValidateProductOutcome.possibleValues(ValidateProductOutcome.java:27)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:55)
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)
	... 22 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateproductOutcome -->
        <test-method is-config="true" signature="endReport()[pri:0, instance:com.Konakart.Testscript.ValidateProductOutcome@4387b79e]" started-at="2019-10-04T21:08:14Z" name="endReport" finished-at="2019-10-04T21:08:20Z" duration-ms="5596" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- com.Konakart.Testscript.ValidateProductOutcome -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
